sys/writeFileSyncBrowser.js

```diff
@@ -2,7 +2,14 @@ const { getFile } = require('./files');
 
 const writeFileSync = (path, data, options) => {
   const file = getFile(path);
+  const { translator } = options;
+
   file.data = data;
+
+  if (translator) {
```

Comment:
I have a hunch there is a cool way to do this all in one or two lines. Do you know how that could be done?

Reply:
How about

const writeFileSync = (path, data, { translator } = {}) => {
  const file = getPath(path);
  file.data = data;
  file.translator = translator;

I guess you could try using Object.assign(file, { data, translator }) but that's getting a bit hard to read, maybe.

Reply:
I like that. It's clear to read and simple